redhat-developer/app-services-cli
# feat(kafka): add interactive prompt for kafka use

### Description
`rhoas kafka use` should prompt a list of available kafka instances.

![Screenshot from 2021-03-29 16-34-37](https://user-images.githubusercontent.com/23582438/112829176-5c2b8f80-90ae-11eb-92ab-eceeca54645a.png)

fixes #496 

### Verification Steps

1. Run `go run ./cmd/rhoas kafka use`
2. It should prompt a list of Kafka instances to select.
3. Navigate using arrow keys and press enter to select an instance.
4. The terminal should output
```
Kafka instance "<name of selected kafka instance>" has been set as the current instance.
```

### Type of change


### Checklist

~~- [ ] Documentation added for the feature~~
